scripts/release.sh

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env sh
+set -e
+PATH=$PATH:$PWD/bin
+GOPATH=$PWD
+
+SRC=${PWD}/src/github.com/SUSE/cf-plugin-backup
+TAG=$(git -C ${SRC} describe --tags --long | awk -F - '{ print $1 }' )
```

Comment:
Instead of the `awk`, we can say `git -C ${SRC} describe --tags --abbrev=0`

Reply:
will do

Reply:
will do

Reply:
Done with commit 26ec7b3de6826d8677420fe18dc7d72b1f188c88